### PR TITLE
Tanstack visual optimizations

### PR DESCRIPTION
**Pull request by nop33**
- Cleans up skeleton loaders
- Fixes entering/exiting animation of charts
- Fixes exiting animations of modals
- Fixes display of address details modal token list

### CODE REVIEW

File: apps/desktop-wallet/src/features/historicChart/HistoricWorthChart.tsx [hunk 1]
```diff
@@ -46,11 +54,14 @@ const startingDates: Record<ChartLength, Dayjs> = {
 }
 
 const HistoricWorthChart = memo(function HistoricWorthChart({
```
Comment (mvaivre) [inline apps/desktop-wallet/src/features/historicChart/HistoricWorthChart.tsx:56]: nit: let's use an arrow function?

File: apps/desktop-wallet/src/features/historicChart/HistoricWorthChart.tsx [hunk 2]
```diff
@@ -46,11 +54,14 @@ const startingDates: Record<ChartLength, Dayjs> = {
...
+  isChartVisible,
+  isChartInitiallyHidden,
```
Comment (mvaivre) [inline apps/desktop-wallet/src/features/historicChart/HistoricWorthChart.tsx:58]: I find those name confusing, it's like they're function returning a boolean.
Maybe we could use `shouldChartBeVisible` & `shouldChartBeInitiallyHidden`, or just `chartVisible` & `chartInitiallyHidden`.

File: apps/desktop-wallet/src/pages/UnlockedWallet/OverviewPage/AmountsOverviewPanel.tsx [hunk 3]
```diff
@@ -26,28 +25,28 @@ import TotalAlphBalance from '@/features/balancesOverview/TotalAlphBalance'
...
-  hidden: { height: 0 }
-}
-
-const AmountsOverviewPanel = ({ className, addressHash, children, showChart }: AmountsOverviewPanelProps) => {
+const AmountsOverviewPanel = ({
+  className,
+  addressHash,
+  children,
+  isChartVisible,
+  isChartInitiallyHidden
```
Comment (mvaivre) [inline apps/desktop-wallet/src/pages/UnlockedWallet/OverviewPage/AmountsOverviewPanel.tsx:47]: Same remark about naming here

**Review by mvaivre — COMMENTED** [review]
Code looks good! Just a few remarks.
I'll test this thoroughly once all the tanstack related PRs are merged.

File: apps/desktop-wallet/src/features/historicChart/HistoricWorthChart.tsx (hunk 2 above)
Comment (nop33) [reply apps/desktop-wallet/src/features/historicChart/HistoricWorthChart.tsx]: Fixed and pushed

File: apps/desktop-wallet/src/features/historicChart/HistoricWorthChart.tsx (hunk 1 above)
Comment (nop33) [reply apps/desktop-wallet/src/features/historicChart/HistoricWorthChart.tsx]: Fixed and pushed

**Review by mvaivre — APPROVED** [review]
LGTM! Thanks for the changes.

**Pull request merged**